docs/outputs.rst

```diff
@@ -150,4 +150,66 @@ I012   ignored            ign_add1
 
 Visual reports
 --------------
-We're working on it.
+Static visual reports can be generated by using the ``--png`` flag when calling 
+tedana from the command line. Images are created and placed within the output 
+directory, in a folder labeled ``figures``. 
+
+These reports consist of three main types of images.
+
+Component Images
+````````````
+.. image:: /_static/example_good_component.png
+  :align: center
+  
+For each component identified by tedana, a single image will be created. Above is an
+example of an accepted component. These are designed for an up-close inspection of both
+the spatial and temporal aspects of the component, as well as ancilliary information. 
```

Comment:
```suggestion
the spatial and temporal aspects of the component, as well as ancillary information. 
```

Reply:
I did it a bit more manually in Atom - better than trying to do it in the browser. (not scared of vim! just away from computer)

Reply:
I'm not a vim fan, myself :laughing: but thanks for updating those line lengths !!